realtime/helper.go

```
@@ -14,7 +13,6 @@ func (l *LiveQuery) addSubscription(id string, c chan *types.SubscriptionEvent)
 	return func() {
 		_, err := l.client.Request(types.TypeRealtimeUnsubscribe, types.RealtimeRequest{Group: l.col, ID: id, Options: l.options})
 		if err != nil {
-			log.Println("Failed to unsubscribe", err)
 		}
```

Simply ignore the error